docs/UserGuide.md

```diff
@@ -45,16 +44,16 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
 **:information_source: Notes about the command format:**<br>
 
 * Words in `UPPER_CASE` are the parameters to be supplied by the user.<br>
-  e.g. in `add n/NAME`, `NAME` is a parameter which can be used as `add n/John Doe`.
+  e.g. in `add n/COMPANY_NAME`, `COMPANY_NAME` is a parameter which can be used as `add n/Meta`.
```

Comment:
Should we change this to `c/` instead? What do the rest think @tienyu2000 @adeearyaa @rylzxc?
If yes, do change it for the other sections in the UG

Reply:
If we were to change it to c/, we could change the parameter name to COMPANY as well since it's quite self explanatory, what do yall think?